tests/suite/model/numbering.typ

```diff
@@ -101,3 +101,15 @@
 --- numbering-negative ---
 // Error: 17-19 number must be at least zero
 #numbering("1", -1)
+
+--- numbering-circled-number ---
```

Comment:
See https://github.com/typst/typst/pull/4036/commits/13cf2f8e7f62d28196e99bb6de618c4302c917be#r1584723939

Reply:
thank you for mentioning!